pg_diffix/node_funcs.h

```diff
@@ -2,7 +2,6 @@
 #define PG_DIFFIX_NODE_FUNCS_H
 
 #include "nodes/params.h"
-#include "nodes/primnodes.h"
```

Comment:
The header references nodes directly and it's good practice to include what you reference in the same file.